src/ceph-disk/ceph_disk/main.py

```diff
@@ -2755,7 +2758,7 @@ def create_key(self):
         bootstrap = self.args.prepare_key_template.format(cluster=cluster,
                                                           statedir=STATEDIR)
         path = self.get_mount_point()
-        secrets = LockboxSecrets(self.args)
+        secrets = LockboxSecrets(self.args, crush_device_class=self.args.crush_device_class)
```

Comment:
nit, i don't think we need to pass `crush_device_class`a separately if it's already in `self.args`.